lld/test/wasm/lto/Inputs/stub2.so

```diff
@@ -0,0 +1,3 @@
+#STUB
+bar:
```

Comment:
This entry is necessary not to crash because `stub-library.s` has a `bar` declaration. Without this we crash with something like `undefined symbol: bar`